src/routes/index.tsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import Signup from '../screens/Signup';
+import EditUser from '../screens/EditUser';
 
 const Routes: React.FC = () => (
   <Router>
     <Switch>
       <Route exact path="/cadastro" component={Signup} />
+      <Route exact path="/editarUsuario" component={EditUser} />
```

Comment:
Acho que a gente pode usar uma parada "mais REST", tipo tudo que for de usuário ficaria no `/usuario` e aí seguia a operação que vai ser de cada página, tipo `/usuario/editar`.

Reply:
Faz sentido